Presentation/UserFunction.cs

```diff
@@ -0,0 +1,34 @@
+using System.Net;
+using System.Text.Json;
+using Domain.Interfaces;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Presentation
+{
+    public class UserFunction
```

Comment:
since we are probably going to have a crud for Users, maybe call this UsersController and make that the main route, what do you think